leaderboardsgg/leaderboard-site
# Re-adding i18n functionality to Nuxt 3

## What

Added i18n functionality back in that works with Nuxt 3, and set Welcom, Login, and Signup buttons to use translation

## Acceptance

### Steps for testing

startup the site in English and see all three in English, and Startup in German and you'll see 2 translated to German and one fallsback to English
